components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -1305,6 +1305,10 @@ private ConfigParameters() {
     public static final String X_WSO2_MUTUAL_SSL = "x-wso2-mutual-ssl";
     public static final String X_WSO2_APP_SECURITY = "x-wso2-application-security";
     public static final String X_WSO2_RESPONSE_CACHE = "x-wso2-response-cache";
+    public static final String X_WSO2_DISABLE_SECURITY = "x-wso2-disable-security";
+    public static final String X_WSO2_THROTTLING_TIER = "x-wso2-throttling-tier";
```

Comment:
Also, I believe this too does not exist in API Manager. However, there is x-throttling-tier.